### DISCUSSION TITLE
ASIO4ALL configuration examples

### DISCUSSION THREAD

**Discussion by npostavs** [jamulussoftware/jamulus#1253] [Community Knowledgebase]
I've collected a few examples of working ASIO4ALL configurations from people I play with, I think it could be helpful for other people to see them. And it could be useful to collect more.

https://github.com/npostavs/jamulus/wiki/ASIO4ALL-configuration-examples

**Comment by ann0see** [comment]
That's great!!! I think you should include it in the official documentation.

**Comment by ann0see** [comment]
Adding mine here too:
![Conexant ISST Audio](https://user-images.githubusercontent.com/20726856/111084554-472df880-8513-11eb-886c-d5d836b7b9f7.png)

**Comment by ann0see** [comment]
@npostavs would you be OK with me linking this as discussion to https://jamulus.io/kb/2021/03/20/ASIO4ALL-Examples.html?

**Reply by npostavs** [reply]
Sure.

**Reply by ann0see** [reply]
Done.
